share/spice/just_delete_me/content.handlebars

```diff
@@ -0,0 +1,15 @@
+{{#if notes}}
+  <p>
+    {{notes}}
```

Comment:
I think it would be good to have a fallback message that explains you should follow the link.

Alternatively, maybe we shouldn't show a result if it has no instructions?

https://beta.duckduckgo.com/?q=delete+instagram+account&t=h&ia=answer -- it's not entirely clear what's supposed to happen here.